Gemfile

```diff
@@ -51,7 +51,7 @@ gem 'redcarpet', platforms: [:ruby]
 gem 'simplecov', require: false
 gem 'slim', '~> 5'
 gem 'yajl-ruby', platforms: [:ruby]
-gem 'zeitwerk', '< 2.7.0' # https://github.com/sinatra/sinatra/issues/2047
+gem 'zeitwerk'
```

Comment:
We should probably try with the old version too, in CI. Hopefully I'll have some time next week if you need help with that.

Reply:
Just pushed that, can you have a look if that's what you had in mind or if you'd rather do it differently?

Reply:
Looks good to me :) I hope to merge this next week